src/main/java/net/atos/zac/signalering/SignaleringenJob.java

```diff
@@ -80,7 +80,7 @@ private void zaakSignaleringenVerzenden() {
         ztcClientService.listZaaktypen(configuratieService.readDefaultCatalogusURI())
                 .forEach(zaaktype -> {
                     final UUID zaaktypeUUID = UriUtil.uuidFromURI(zaaktype.getUrl());
```

Comment:
Zaaktype heeft een convenience method: getUUID() Dan is die UriUtil.uuidFromURI()) niet nodig.